lib/modules/datasource/hex/schema.ts

```diff
@@ -0,0 +1,47 @@
+import { z } from 'zod';
+import { LooseArray } from '../../../util/schema-utils';
+import type { Release, ReleaseResult } from '../types';
+
+export const HexRelease = z
+  .object({
+    html_url: z.string().optional(),
+    meta: z
+      .object({
+        links: z.object({
+          Github: z.string(),
+        }),
+      })
+      .nullable()
+      .catch(null),
+    releases: LooseArray(
+      z.object({
+        version: z.string(),
+        inserted_at: z.string().optional(),
+      }),
+    ).refine((releases) => releases.length > 0, 'No releases found'),
```

Comment:
what is this doing? 🤔

Reply:
We need to return `null` for empty array of releases (probably, never happens)